ci/script.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -eEuo pipefail
+
+echo "=== Local variables ==="
```

Comment:
To be honest with you I do not see value of having this ECHO statement since we are not even printing anything with it :shrug: